src/bindings/c/ov/include/c_api/ov_c_api.h

```diff
@@ -0,0 +1,1122 @@
+                                                        ov_preprocess_output_tensor_info_t **preprocess_output_tensor_info);
+
+/**
+ * @brief Release the memory allocated by ov_preprocess_output_tensor_info_t.
+ * @param preprocess_output_tensor_info A pointer to the ov_preprocess_output_tensor_info_t to free memory.
+ */
+OPENVINO_C_API(void) ov_preprocess_output_tensor_info_free(ov_preprocess_output_tensor_info_t *preprocess_output_tensor_info);
+
+/**
+ * @brief Set ov_preprocess_input_tensor_info_t precesion. 
+ * @param preprocess_output_tensor_info A pointer to the ov_preprocess_output_tensor_info_t.
+ * @param element_type A point to element_type
+ */
+OPENVINO_C_API(ov_status_e) ov_preprocess_output_set_element_type(ov_preprocess_output_tensor_info_t* preprocess_output_tensor_info,
+                                                        const ov_element_type_e element_type);
+
+/**
+ * @brief Get current input model information.
+ * @param preprocess_input_info A pointer to the ov_preprocess_input_info_t.
+ * @param preprocess_input_model_info A pointer to the ov_preprocess_input_model_info_t
+ * @return Status code of the operation: OK(0) for success.
+*/
+OPENVINO_C_API(ov_status_e) ov_preprocess_input_get_model_info(ov_preprocess_input_info_t* preprocess_input_info,
+                                                        ov_preprocess_input_model_info_t **preprocess_input_model_info);
+
+/**
+ * @brief Release the memory allocated by ov_preprocess_input_model_info_t.
+ * @param preprocess_input_model_info A pointer to the ov_preprocess_input_model_info_t to free memory.
+ */
+OPENVINO_C_API(void) ov_preprocess_input_model_info_free(ov_preprocess_input_model_info_t *preprocess_input_model_info);
+
+/**
+ * @brief Set layout for model's input tensor. 
+ * @param preprocess_input_model_info A pointer to the ov_preprocess_input_model_info_t
+ * @param layout A point to ov_layout_t
+ */
+OPENVINO_C_API(ov_status_e) ov_preprocess_input_model_set_layout(ov_preprocess_input_model_info_t* preprocess_input_model_info,
+                                                        const ov_layout_t layout);
+
+/**
+ * @brief Adds pre/post-processing operations to function passed in constructor. 
+ * @param preprocess A pointer to the ov_preprocess_t.
+ * @param model A pointer to the ov_model_t.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e) ov_preprocess_build(const ov_preprocess_t* preprocess,
+                                            ov_model_t **model);
+
+/**
+ * @brief Gets runtime model information from a device.
+ * @param compiled_model A pointer to the ov_compiled_model_t.
+ * @param model A pointer to the ov_model_t.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e) ov_compiled_model_get_runtime_model(const ov_compiled_model_t* compiled_model,
```

Comment:
I suppose this method is not needed for C API.
It's mostly for profiling, while C is used for deployment scenarios.